test/unit/controllers/declaration/AdditionalFiscalReferencesControllerSpec.scala

```diff
@@ -55,9 +53,9 @@ class AdditionalFiscalReferencesControllerSpec extends ControllerSpec {
     "return 200 (OK)" when {
 
       "display page method is invoked with empty cache" in new SetUp {
-        val result = controller.displayPage("itemId")(getRequest())
+        val result: Future[Result] = controller.displayPage("itemId")(getRequest())
 
-        status(result) must be(OK)
+        status(of = result) must be(OK)
```

Comment:
what's this?

Reply:
me being stupid with intellij and trying to reduce warnings.